internal/convertor/device/device.go

```diff
@@ -115,7 +132,7 @@ func (d *Device) GenerateOpenconfig() error {
 	// Assemble global configuration
 	bgpKey := openconfig.NetworkInstance_Protocol_Key{Identifier: openconfig.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, Name: "bgp"}
 
-	config := openconfig.Device{
+	d.Config.Openconfig = &openconfig.Device{
```

Comment:
before your PR, we did not store the config and the JSON until we are sure it has been validated.
Any reason to change that?

Reply:
yes, left over of the previous version